lxd/resources.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
```

Comment:
Missing separator between external and internal imports.

Reply:
added